ESCPOS_NET/Emitters/BaseCommandEmitter/ImageCommands.cs

```diff
@@ -118,5 +123,23 @@ public virtual byte[] PrintImage(byte[] image, bool isHiDPI, bool isLegacy = fal
                 return ByteSplicer.Combine(SetImageDensity(isHiDPI), BufferImage(image, maxWidth, isLegacy, color), WriteImageFromBuffer());
             }
         }
+
+        public virtual byte[] PrintRasterizedText(string text, FontFamily fontFamily, float size, FontStyle fontStyle = FontStyle.Regular)
+        {
+            var font = fontFamily.CreateFont(size, fontStyle);
+
+            var options = new RendererOptions(font);
+            var rect = TextMeasurer.Measure(text, options);
+
+            Image image = new Image<Rgba32>((int)rect.Width, (int)rect.Height);
+            image.Mutate(x => x.DrawText(text, font, SixLabors.ImageSharp.Color.Black, new PointF(0, 0)));
+
+            var memoryStream = new MemoryStream();
+            image.SaveAsPng(memoryStream);
+            var bytes = memoryStream.ToArray();
+            memoryStream.Dispose();
```

Comment:
In Java, there is garbage collection that can usually handle such cleanup. Just curious how this works in C#.

Reply:
My understanding is that—as we're explicitly working with memory here—we need to make sure we dispose of the intermediary object …but I might be wrong!